### DISCUSSION TITLE
License model for the outputs generated by SystemRDL tools

### DISCUSSION THREAD

**Discussion by CodeStillBroken** [SystemRDL/systemrdl-compiler#184] [Q&A] [answered]
Hi, 

Can your team make a clear and unambiguous statement on the PeakRDL website ( e.g. in the licensing section ) clarifying what license, if any, applies to the output generated by running the PeakRDL tools covered by GPL v3?    My understanding is that the output of the program is not governed by the terms of the program's license.  If that is the case, it would be helpful to clearly state that the customer owns all rights to the outputs generated by the PeakRDL tools on your website :) 

Thanks.

**Comment by amykyta3** [comment]
Will do!
That is definitely the case for all output generated for the core PeakRDL tools. I have some statements in the [regblock generator docs](https://peakrdl-regblock.readthedocs.io/en/latest/licensing.html#what-is-exempt-from-the-gpl-v3-license), but I agree it would be good to make a general statement on this in the main docs too.

I'm going to transfer this to be an issue ticket for that project so I don't lose track of this.

**Reply by amykyta3** [reply]
See: https://github.com/SystemRDL/PeakRDL/issues/33

**Comment by amykyta3** [comment, marked as answer]
I realized I already have some text that clearly states this. See: https://peakrdl.readthedocs.io/en/latest/licensing.html#what-is-exempt-from-the-gpl-v3-license
